exercises/elm-2-b/README.md

```diff
@@ -0,0 +1,23 @@
+## Introduction
+
+This is Part 1 of our research into how people write Elm differently.
+This exercise should take 10-15 minutes for a programmer who is proficient in Elm.
+There is no right or wrong way to approach it - just do what feels the most natural to you.
+Feel free to use books, Google or Stack Overflow,
+just like you would if you were programming normally,
+but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+Well done on the printed menu analysis!
```

Comment:
For the instruction, we usually repeat what the previous instructions were, as these are the instructions that are implemented by the starting code. For an example, see https://github.com/exercism/research_experiment_1/tree/master/exercises/elm-2-a#instructions.

Reply:
Ok, I've changed this in 51682d5